src/test/completion.test.ts

```diff
@@ -69,15 +69,13 @@ suite('HTML Completion', () => {
 
 		testCompletionFor('<input t|ype="text"', {
 			items: [
-				{ label: 'type', resultText: '<input type="text"' },
 				{ label: 'tabindex', resultText: '<input tabindex="text"' }
 			]
 		});
 
 		testCompletionFor('<input type="text" |', {
 			items: [
 				{ label: 'style', resultText: '<input type="text" style="$1"' },
-				{ label: 'type', resultText: '<input type="text" type="$1"' },
```

Comment:
Use `notAvailable: true` to make sure the attribute proposal is not in the result